.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,20 @@
-### What changes were proposed in this pull request?
+### What is the goal of this PR?
+
+
+### Links (JIRA, Notion, Miro)
```

Comment:
JIRA ticket should be linked automatically by extracting JIRA ticket key from PR name (I'm not sure that it works correctly from github side because I didn't see any links in my last PRs). And this naming convention should be probably documented in [contributors guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors).

But there can also be some other related JIRA tickets, so yeah, makes sense.